### PR TITLE
RavenDB-16617 Creating a dedicated endpoint for proxy to be sure that…

### PR DESCRIPTION
**Pull request by arekpalinski**
… we'll get info about databases that are accessible using the certificate using the orginal connection

### CODE REVIEW

**Review by ppekrol — APPROVED** [review]

File: src/Raven.Server/NotificationCenter/Handlers/ClusterDashboardHandler.cs [hunk 1]
```diff
@@ -75,34 +75,101 @@ public async Task Get()
...
                 {
-                    if (Logger.IsInfoEnabled)
-                        Logger.Info("Error encountered in cluster dashboard handler", ex);
+                    await HandleException(ex, webSocket);
+                }
+            }
+        }
 
-                    try
+        [RavenAction("/cluster-dashboard/remote/watch", "GET", AuthorizationStatus.ClusterAdmin, SkipUsagesCount = true)]
```
Comment (ayende) [inline src/Raven.Server/NotificationCenter/Handlers/ClusterDashboardHandler.cs]: Need `admin` in the link, no?

Comment (ayende) [reply src/Raven.Server/NotificationCenter/Handlers/ClusterDashboardHandler.cs]: What happens if we aren't secured?

Comment (ppekrol) [reply src/Raven.Server/NotificationCenter/Handlers/ClusterDashboardHandler.cs]: IIRC everyone is cluster admin in unsecured?

Comment (arekpalinski) [reply src/Raven.Server/NotificationCenter/Handlers/ClusterDashboardHandler.cs]: 1. Added `admin` in https://github.com/ravendb/ravendb/pull/12106/commits/36c23bea0dd0abcccd10625f9588b51b8260a40d
2. Correct. Then we're ClusterAdmin by default

**Pull request merged**